music-quiz/src/views/scoreboardView.js

```diff
@@ -0,0 +1,38 @@
+import trophy from "../images/trophy.png";
+import { Image, ListGroup, Badge } from "react-bootstrap";
+import { Link } from "react-router-dom";
+
+export default function ScoreboardView(props) {
+  let sortedScores = Object.values(props.scores).sort(
+    (a, b) => a.score - b.score
```

Comment:
For me it sorts in ascending order (lowest to highest), but what we want is highest to lowest right?

Reply:
Yes, we flipped it. This is now fixed